cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,81 @@
+/* eslint-disable max-len */
+/* eslint-disable indent */
+/* eslint-disable cypress/no-unnecessary-waiting */
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+  });
+  it('should contain pagination', () => {
+    cy.visit('https://demoqa.com/webtables');
+
+    cy.get('.-previous').should('contain', 'Previous');
```

Comment:
It seems you can't test the pagination simply checking if the buttons are there.